src/MetadataStorage.ts

```diff
@@ -26,11 +27,21 @@ export interface RepositoryMetadata {
   entity: Function;
 }
 
+export interface Config {
```

Comment:
I think that `Config` is too ambiguous. Wouldn't `RuntimeConfiguration` or `GlobalConfiguration` be better?

Reply:
Fair point, I agree. I'm not sure either of those accurately describe what the config is for though. I did think maybe something like `FireORMConfig` at first but it felt weird to reference the project name in the code...  Maybe something like `InitializeOptions`?

Reply:
Yeah, that can work.

Reply:
Or maybe `MetadataStorageConfig`, seeing as it's the config property for that class

Reply:
That works too.